homeassistant/components/lutron_caseta/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "lutron_caseta",
   "name": "Lutron Cas\u00e9ta",
   "documentation": "https://www.home-assistant.io/integrations/lutron_caseta",
-  "requirements": ["pylutron-caseta==0.16.0"],
+  "requirements": ["pylutron-caseta==0.17.1"],
```

Comment:
Does the dep need to be bumped for this PR?

Reply:
No, just for the Area naming fix.  I can do it after.
Fine to include here too though.

Reply:
This one is already rather large and difficult review + test so I'd prefer to leave that for another PR.

Reply:
agreed